### ISSUE TITLE
Pill design

### ISSUE THREAD

**Issue by technimadhu** [alexandernst/angular-multi-select#52]
Can you support ability to clear a selection from the header (with an x next to selection)
![capture](https://cloud.githubusercontent.com/assets/7958275/10719703/084e10dc-7b64-11e5-8bc1-62fcf9ac13db.PNG)

**Comment by alexandernst** [comment]
Yes, I'll probably start v6 on Wednesday, will keep this FR in mind.
